packages/suite/src/views/onboarding/index.tsx

```diff
@@ -66,9 +75,11 @@ const Onboarding = ({ prerequisite, modal }: InjectedModalApplicationProps) => {
         }
     }, [activeStepId]);
 
+    const allowedModal = useModalAllowed();
```

Comment:
No, because it returns modal data instead of boolean. Nevertheless resolved by https://github.com/trezor/trezor-suite/pull/5313/commits/6915d6c06ded6fb8b95ccffe9d2c6831f301d38b